<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PomodroidoLiteMain</class>
 <widget class="QMainWindow" name="PomodroidoLiteMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PomodroidoLite</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_pic_display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>951</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pic/pomodroPicNew.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QChartView" name="widget_3DBars" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>711</width>
      <height>471</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>390</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/pic/historyoftomatoes2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>240</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_create">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>420</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
    </property>
    <property name="text">
     <string>创建🍅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_discard">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>420</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
    </property>
    <property name="text">
     <string>丢掉🍅</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set_goal">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>370</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
    </property>
    <property name="text">
     <string>设定🍅</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pic/pic1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_diary">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>320</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
    </property>
    <property name="text">
     <string>写日志</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_database">
    <property name="title">
     <string>🍅仓库（Database）</string>
    </property>
    <addaction name="action_databaseQuery"/>
   </widget>
   <widget class="QMenu" name="menu_statistics">
    <property name="title">
     <string>小工具（Gadget）</string>
    </property>
    <addaction name="action_timeCountDown"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>视图（View）</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>关于（About）</string>
    </property>
    <addaction name="action_Version_Info"/>
   </widget>
   <addaction name="menu_database"/>
   <addaction name="menu_statistics"/>
   <addaction name="menu_view"/>
   <addaction name="menu_about"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Version_Info">
   <property name="text">
    <string>软件介绍(Soft Intro)</string>
   </property>
  </action>
  <action name="action_databaseQuery">
   <property name="text">
    <string>历史番茄查询(Query)</string>
   </property>
  </action>
  <action name="action_timeCountDown">
   <property name="text">
    <string>倒计时(Countdown)</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>简易计算器(Calculator Lite)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_set_goal</sender>
   <signal>clicked()</signal>
   <receiver>PomodroidoLiteMain</receiver>
   <slot>setTomatoSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>880</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>950</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_create</sender>
   <signal>clicked()</signal>
   <receiver>PomodroidoLiteMain</receiver>
   <slot>createTomatoSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>950</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_discard</sender>
   <signal>clicked()</signal>
   <receiver>PomodroidoLiteMain</receiver>
   <slot>discardTomatoSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>906</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>947</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setTomatoSlot()</slot>
  <slot>createTomatoSlot()</slot>
  <slot>discardTomatoSlot()</slot>
 </slots>
</ui>
